code/__HELPERS/mobs.dm

```diff
@@ -127,11 +127,12 @@ Proc for attack log creation, because really why not
 
 /proc/add_logs(mob/user, mob/target, what_done, object=null, addition=null)
 	var/newhealthtxt = ""
-	var/turf/attack_location = get_turf(target)
-	var/coordinates = "([attack_location.x],[attack_location.y],[attack_location.z])"
+	var/coordinates = ""
 	if (target && isliving(target))

```

Comment:
this won't log anything if the target is a ghost. Are we sure add_logs() is never called with a ghost target?